README.md

```diff
@@ -1,76 +1,101 @@
-# HTML5 Media
+# Working with HTML5 Media
+
+## Problem Statement
+The internet is a highly interactive environment. Even blogs where text
+may be the most dominant content, audio and video elements often support,
+or even supplement, the content. In the case of 3rd party media sites such
+as YouTube, SoundCloud, and Vimeo, you are given an all-in-one line of code
+with configurable properties that creates the embeddable content for you--
+but how do you do this manually in your application? Often times, you will be working
+with, and manipulating, media that is being uploaded into or generated by your
+app. HTML has evolved dramatically since the days of the generic `embed` tag.
+How is this done today?
 
 ## Overview
+1. Demonstrate how to embed audio elements in HTML5
+2. Demonstrate how to embed video elements in HTML5
+3. Link to audio and video converters
+
+## Demonstrate How to Embed Audio Elements in HTML5
+Embedding audio can be a nice touch to a website. If you had a MySpace page back
+in the 2000s, you can probably to this. HTML was very different back
+then. The HTML tags were not semantic, especially not for media, and it was less 
+forgiving to install. If you wanted to embed audio, you also needed to embed an 
+audio  player, which might have to be flash, java, or javascript based. If you did 
+not, you could mistakenly start audio with no way for the user to control it. 
+This is no longer an issue, as HTML5 has its own default media players supported 
+in the browser.
```

Comment:
1. I'd reduce this paragraph to its strongest points.

> Since the browsers were primarily document-oriented, the 90's and 2000's saw many commercial media players develop: Shockwave, Flash, etc. The W3C wanted the web to remain open. If one of those providers achieved market dominance, they could force everyone onto their platform ("The Internet, powered by Adobe Flash"). To avoid this, HTML5 provided us our first media solutions that weren't attached to a company.

2. Also, observe sensible capitalization rules: Flash, Java, JavaScript. Probably preface with company names so that readers know these are commerce words.